src/EndpointManagerLauncher/endpointmanagerlauncher.csproj

```diff
@@ -19,6 +19,14 @@
     <ApplicationIcon>..\resources\icons\BridgeToKubernetes_VS_NoHalo.ico</ApplicationIcon>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702;8002</NoWarn>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">

```

Comment:
@Eneuman I found many links that is asking for supressing CS1701, CS1702 warnings from dotnet compilers. But I couldn't find the exact issue or link for it. Can you paste the link as comment here so that in future someone will know it. Thanks.

Reply:
@hsubramanianaks I assumed that this was the default behavior because of how VS behaves.
If you remove the property groups so that the .csproj file does not have a <NoWarn> tag, VS GUI will look like this:

![image](https://github.com/Azure/Bridge-To-Kubernetes/assets/12099277/9e25f3c4-85b3-4d0c-a540-91083c949ed1)

To me this indicates that for that specific project type, it will ignore CS1701, CS1702 by default, but maby thats a false asumption?

Reply:
@Eneuman Nope I agree VS GUI does that. But my point was is there GH issue or link why we are ignoring these warning for codes 1701,1702. if there is one please add them as comment in this file ?

Reply:
I coudn't find any issue or link describing this VS behaviour so I reverted this changed :)